src/components/ui/project/irysmanga/modal-components/ModalTabLicences.tsx

```diff
@@ -8,40 +8,44 @@ export default function ModalTabLicenses() {
 	const modalDataRoot = getModalDataRoot(manga);
 	const modalData = getLocalisedModalData(modalDataRoot, readerLanguage);
 	const { t } = useTranslation('reader', 'modal-licenses');
+
+	const coverGuidelinesParts = modalData.artLicensesContent.split(modalData.coverGuidelines);
+	const additionalInfoParts = modalData.additionalInfoContent.split(modalData.contactInfo);
```

Comment:
Hm, just to check, is this correct? Splitting removes the delimiter/separator from the returned array IIRC.

Asking as the page shows an error right now on this branch so I don't know if it's correct or not.

Reply:
Yep that's correct. The delimiter is the text we want to replace with a link, so we're splitting the string to get everything before and after it.